### PR TITLE
Add the Hank project to Portland 2023 Writing Day page

### PR DESCRIPTION
**Pull request by kaycebasques**
----
:books: Documentation preview :books:: https://writethedocs-www--1942.org.readthedocs.build/

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]
👌

File: docs/conf/portland/2023/writing-day.rst [hunk 1]
```diff
@@ -201,3 +201,23 @@ This sprint will give you a few options:
 
 The documentation is written in Sphinx & reStructuredText, but you can try out 
 your own project using any framework, as long as it's open source.
+
+Hank
+~~~~
+
```
Comment (rosewms) [inline docs/conf/portland/2023/writing-day.rst]: @kaycebasques simplified the title

Comment (rosewms) [reply docs/conf/portland/2023/writing-day.rst]: @kaycebasques, I also reformatted the information below for easier reading/skimming. If you're good with it, I'll add it to the site.

I also may move it to the top of the project list, we're adding new projects at the top. I have two other PRs in so I'll make the change after those are approve.

Thanks for your submission!

**Comment by rosewms** [comment]
Direct link to preview: https://writethedocs-www--1942.org.readthedocs.build/conf/portland/2023/writing-day/

File: docs/conf/portland/2023/writing-day.rst (hunk 1 above)
Comment (kaycebasques) [reply docs/conf/portland/2023/writing-day.rst]: Thanks for the review. You can edit everything however you see fit.

**Review by rosewms — APPROVED** [review]
Edited order, merging shortly.

**Pull request merged**